development/browser_test/Dockerfile

```diff
@@ -11,6 +11,8 @@ COPY ./composer.lock /data/
 COPY ./run.sh /data/
 
 RUN chmod a+x /data/run.sh
+RUN composer self-update --no-interaction
+RUN composer update --no-interaction
```

Comment:
We don't normally have a `composer update` call in our Dockerfile... that would update dependencies on each build, rather than simply installing the dependencies that we've already defined in our composer.lock file (whenever we manually updated most recently).

I'm not necessarily against this change, it's just atypical for us. Did this fix a particular problem, or was it to reduce the number of manual updates needed when doing the browser tests? (Or some other reason?)

Thanks.

Reply:
I was doing various types of troubleshooting. I don't remember really why I included these. I'll try without them.